src/current/v24.2/changefeed-sinks.md

```diff
@@ -212,6 +212,54 @@ See the [Changefeed Examples]({% link {{ page.version.version }}/changefeed-exam
 
 {% include {{ page.version.version }}/cdc/note-changefeed-message-page.md %}
 
+## Amazon MSK
+
+{{site.data.alerts.callout_info}}
+On CockroachDB {{ site.data.products.core }} clusters, you must create instances in the same VPC as the MSK or MSK Serverless cluster in order for the changefeed to authenticate successfully.
+
+If you would like to connect a CockroachDB {{ site.data.products.dedicated }} cluster to an Amazon MSK cluster, contact your Cockroach Labs account team.
+{{site.data.alerts.end}}
+
+Changefeeds can deliver messages to Amazon MSK clusters ([Amazon Managed Streaming for Apache Kafka](https://docs.aws.amazon.com/msk/latest/developerguide/what-is-msk.html)). Amazon MSK cluster types include: [MSK](https://docs.aws.amazon.com/msk/latest/developerguide/create-cluster.html) and [MSK Serverless](https://docs.aws.amazon.com/msk/latest/developerguide/serverless.html). Changefeeds support the following authentication methods for these MSK cluster types:
+
+- MSK: `SCRAM` or `IAM`
+- MSK Serverless: `IAM`
+
+{% include_cached new-in.html version="v24.2" %} Changefeeds can deliver messages to MSK and MSK Serverless clusters using AWS IAM roles.
+
+{% comment %}will change out the links here to CRDB tutorials in a follow-up PR.{% endcomment %}
+
+For initial setup guides, refer to the AWS documentation:
+
+- [Getting started using Amazon MSK](https://docs.aws.amazon.com/msk/latest/developerguide/getting-started.html)
```

Comment:
Minor comment: I usually avoid naming external articles since they can change. I feel it's simpler to maintain something like "refer to the AWS documentation for [MSK](link) or [MSK Serverless](link) clusters". But it's obviously less clear than what you have currently.